pyproject.toml

```diff
@@ -83,6 +84,7 @@ pytest-watcher = "^0.2.6"
 freezegun = "^1.2.2"
 responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
+gptcache = ">=0.1.7"
```

Comment:
This should be an optional dependency since we are lazily loading the import

Reply:
Fixed. Thanks for the heads-up.